dev/managedservicesplatform/spec/monitoring.go

```diff
@@ -1,14 +1,22 @@
 package spec
 
 import (
-	"time"
+	"strings"
 
 	"github.com/grafana/regexp"
 
 	"github.com/sourcegraph/sourcegraph/lib/errors"
 )
 
 var codeClassPattern = regexp.MustCompile(`\dx+`)
+var customAlertNamePattern = regexp.MustCompile(`^[-A-Za-z0-9 ]+$`)
+
+type SeverityLevel string
+
+const (
+	SeverityLevelWarning  SeverityLevel = "WARNING"
+	SeverityLevelCritical SeverityLevel = "CRITICAL"
+)
```

Comment:
There's a _lot_ of stuff in this package, so I suggest being more verbose in the naming here, e.g.

```suggestion
// AlertSeverityLevel is used for configuration notification urgency for monitoring alerts.
type AlertSeverityLevel string

const (
	AlertSeverityLevelWarning  AlertSeverityLevel = "WARNING"
	AlertSeverityLevelCritical AlertSeverityLevel = "CRITICAL"
)
```

This was less than an issue before when this type was in a smaller package